app/assets/stylesheets/layout.scss

```diff
@@ -0,0 +1,9 @@
+.top-bar{
+  background-color: black;
+}
+html {
+  background: url("dive1.jpeg") no-repeat center center fixed;
```

Comment:
I suggest changing `url(` here to `image-url(` which may help with issues in production